ipynb/tests/Frequency_Invariance_Test.ipynb

```diff
@@ -0,0 +1,799 @@
+      "2016-12-13 18:45:25,427 INFO    : Executor     : Experiment 4/5, [freq_1100000:fie_10pct] 1/1\n",
+      "2016-12-13 18:45:25,842 INFO    : Executor     : Freezing all tasks except: init,systemd,sh,ssh\n",
+      "2016-12-13 18:45:29,260 WARNING : Executor     : FTrace events collection enabled\n",
+      "2016-12-13 18:45:35,878 INFO    : Workload     : Workload execution START:\n",
+      "2016-12-13 18:45:35,879 INFO    : Workload     :    /home/brendan/devlib-target/bin/taskset 0x2 /home/brendan/devlib-target/bin/rt-app /home/brendan/devlib-target/run_dir/fie_10pct_00.json 2>&1\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2016-12-13 18:45:46,348 INFO    : Executor     : Collected FTrace binary trace:\n",
+      "2016-12-13 18:45:46,350 INFO    : Executor     :    <res_dir>/rtapp:freq_1100000:fie_10pct/1/trace.dat\n",
+      "2016-12-13 18:45:46,351 INFO    : Executor     : Collected FTrace function profiling:\n",
+      "2016-12-13 18:45:46,352 INFO    : Executor     :    <res_dir>/rtapp:freq_1100000:fie_10pct/1/trace_stat.json\n",
+      "2016-12-13 18:45:46,353 INFO    : Executor     : Un-freezing userspace tasks\n",
+      "2016-12-13 18:45:48,864 INFO    : Executor     : --------------------------------------------------------------------------------\n",
+      "2016-12-13 18:45:48,866 INFO    : Executor     : \n",
+      "2016-12-13 18:45:48,867 INFO    : Executor     : ################################################################################\n",
+      "2016-12-13 18:45:48,868 INFO    : Executor     : Experiments execution completed\n",
+      "2016-12-13 18:45:48,869 INFO    : Executor     : ################################################################################\n",
+      "2016-12-13 18:45:48,870 INFO    : Executor     : Results available in:\n",
+      "2016-12-13 18:45:48,872 INFO    : Executor     :       /home/brejac01/sources/lisa/results/20161213_184321\n"
+     ]
+    }
+   ],
+   "source": [
+    "t.setUpClass()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Show variance in util_avg and load_avg\n",
+    "We want to see the same util_avg and load_avg values regardless of frequencies - the bar charts below should have bars all with roughly the same height."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Get the frequency an experiment was run at\n",
+    "def experiment_freq(exp):\n",
+    "    [cpu] = exp.wload.cpus\n",
+    "    freq = exp.conf['cpufreq']['freqs'][cpu]\n",
+    "    return freq\n",
+    "freqs = [experiment_freq(e) for e in t.executor.experiments]"
```

Comment:
Yeah, I initially didn't do that because I heard $colleague bemoaning the number of aliases LISA has and didn't want to add another! But I think in this case if we can document the `LisaTest` API properly then it would be useful to actually hide `Executor` completely.